compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.uikit.kt

```diff
@@ -372,7 +375,9 @@ internal class MetalRedrawer(
                     commandBuffer.label = "Present"
 
                     if (!presentsWithTransaction) {
-                        commandBuffer.presentDrawable(metalDrawable)
+                        // scheduleDrawablePresentation consumes metalDrawable
+                        // don't use metalDrawable after this call
```

Comment:
Can we just nullify it?

Reply:
It's a smart cast `val`, doing so will require to put force-unwraps everywhere